src/main/java/org/fenixedu/learning/domain/executionCourse/ExecutionCourseSite.java

```diff
@@ -56,13 +57,15 @@
     public ExecutionCourseSite(ExecutionCourse executionCourse) {
         checkNotNull(executionCourse);
         setExecutionCourse(executionCourse);
-        setPublished(true);
+
         setFolder(folderForPath(PortalConfiguration.getInstance().getMenu(), "courses"));
         setSlug(on("-").join(getExecutionCourse().getSigla(), getExecutionCourse().getExternalId()));
-        setCanAdminGroup(NobodyGroup.get());
-        setCanPostGroup(NobodyGroup.get());
-        setBennu(Bennu.getInstance());
 
+        setCreationDate(new DateTime());
+        setCanAdminGroup(DynamicGroup.get("managers"));
```

Comment:
I have some concerns settings this to manager. All the managers will be have access to any execution course page.